src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -253,6 +262,271 @@ static bool CollectCrashDump(Process process, string path)
             return fSuccess && createdump.ExitCode == 0;
         }
 
+        private static List<string> knownNativeModules = new List<string>() { "libcoreclr.so", "libclrjit.so" };
+        private static string TO_BE_CONTINUE_TAG = "<TO_BE_CONTINUE>";
+        private static string SKIP_LINE_TAG = "# <SKIP_LINE>";
+
+
+        static bool RunProcess(string fileName, string arguments)
+        {
+            Process proc = new Process()
+            {
+                StartInfo = new ProcessStartInfo()
+                {
+                    FileName = fileName,
+                    Arguments = arguments,
+                    UseShellExecute = false,
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                }
+            };
+
+            Console.WriteLine($"Invoking: {proc.StartInfo.FileName} {proc.StartInfo.Arguments}");
+            proc.Start();
+
+            Task<string> stdOut = proc.StandardOutput.ReadToEndAsync();
+            Task<string> stdErr = proc.StandardError.ReadToEndAsync();
+            if(!proc.WaitForExit(DEFAULT_TIMEOUT_MS))
+            {
+                proc.Kill(true);
+                Console.WriteLine($"Timedout: '{fileName} {arguments}");
+                return false;
+            }
+
+            Task.WaitAll(stdOut, stdErr);
+            string output = stdOut.Result;
+            string error = stdErr.Result;
+            if (!string.IsNullOrWhiteSpace(output))
```

Comment:
I was suggesting you could just delete the if condition. There doesn't seem much harm in writing an empty output, and the output should never be null

Reply:
Ah I see.